### ISSUE TITLE
Unable to see test results with a screenreader

### ISSUE THREAD

**Issue by webczat** [formulahendry/vscode-dotnet-test-explorer#243]
I am using a screenreader. I can see tests in a tree view, but I don't seem to have any info about their results there. Not sure if it's even displayed there, or that is really not shown to a screenreader.
